### PR TITLE
.github: Add settings.yml and CODEOWNERS to manage repo via PR

### PR DESCRIPTION
**Pull request by justaugustus**
- .github: Add default CODEOWNERS and settings.yml
- .github: Update CODEOWNERS/settings.yml to reflect maintainers
- .github: Add @OliverFendt as a maintainer

### CODE REVIEW

File: .github/settings.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  # maintain
+  - username: misappi
+    permission: maintain
+
+  # push
+  - username: cornelius
+    permission: push
+
+  - username: mkurzman
+    permission: push
```
Comment (anajsana) [inline .github/settings.yml:35]: @justaugustus Could you please add @OliverFendt as a collaborator? 🙂 
```
- username: oliverfendt
  permission: push
```

Comment (anajsana) [reply .github/settings.yml:35]: Also, all these contributors (@misappi @cornelius @mkurzman and @OliverFendt ) should have merge permissions. Not sure if `push` permission also allows merging PRs in this repo, or should be upgraded to `maintain`

**Review by anajsana — CHANGES_REQUESTED** [review]

**Comment by anajsana** [comment]
Thanks! LGTM 👍

File: .github/settings.yml (hunk 1 above)
Comment (justaugustus) [reply .github/settings.yml]: > Also, all these contributors (@misappi @cornelius @mkurzman and @OliverFendt ) should have merge permissions. Not sure if `push` permission also allows merging PRs in this repo, or should be upgraded to `maintain`

`push` is sufficient for write access, but I've upgraded everyone to `maintain` to allow maintenance of the repo w/o too much SC oversight.

> @justaugustus Could you please add @OliverFendt as a collaborator? 🙂

He wasn't listed before (see below), but I've added him now!

Screen Shot 2022-01-18 at 07 36 27

**Comment by justaugustus** [comment]
> Thanks! LGTM 👍

Awesome! Merging...

**Pull request merged**